src/app/clusters/level-control/level-control.cpp

```diff
@@ -238,18 +238,13 @@ class DefaultLevelControlSceneHandler : public scenes::DefaultSceneHandlerImpl
 
         // TODO : Implement action on frequency when frequency not provisional anymore
         // if(LevelControlHasFeature(endpoint, LevelControl::Feature::kFrequency)){}
-        Status status;
+
         CommandId command = LevelControlHasFeature(endpoint, LevelControl::Feature::kOnOff) ? Commands::MoveToLevelWithOnOff::Id
                                                                                             : Commands::MoveToLevel::Id;
 
-        status = moveToLevelHandler(endpoint, command, level, app::DataModel::MakeNullable(static_cast<uint16_t>(timeMs / 100)),
-                                    chip::Optional<BitMask<OptionsBitmap>>(), chip::Optional<BitMask<OptionsBitmap>>(),
-                                    INVALID_STORED_LEVEL);
-
-        if (status != Status::Success)
-        {
-            return CHIP_ERROR_READ_FAILED;
-        }
+        moveToLevelHandler(endpoint, command, level, app::DataModel::MakeNullable(static_cast<uint16_t>(timeMs / 100)),
```

Comment:
But "level" here can't be a null value, right?  It's actually 255 if the value was null?

It would be better if we explicitly didn't try to move to level if it's null instead of relying on moveToLevelHandler doing some sort of check on values that it's never supposed to see.

Reply:
Correct, null would be 255, which would end up out of bounds for the moveToLevelHandler.  I'll explicitely ignore it instead.

Reply:
Added check for null value.